packages/node_modules/@ciscospark/spark-core/src/config.js

```diff
@@ -73,8 +73,11 @@ export default {
     preDiscoveryServices: {
       hydraServiceUrl: process.env.HYDRA_SERVICE_URL || 'https://api.ciscospark.com/v1'
     },
+    validateDomains: process.env.VALIDATE_DOMAINS,
```

Comment:
you should add a `|| false` here too

Reply:
If you are worried about `undefined` here, do `!!process.env.VALIDATE_DOMAINS`

Reply:
its a string for now added  || "false" . any suggestion on how to do it . I dont think we can pass boolean

Reply:
I did something similar for [skipping flaky tests](https://github.com/webex/spark-js-sdk/blob/master/packages/node_modules/@ciscospark/test-helper-mocha/src/index.js#L60). What's the best way to accomplish this?